cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,12 @@
-/// <reference types="cypress" />
-
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('input[type="email"]').click();
+    cy.get('input[type="email"]').type('mlemeshev@i.ua');
+    cy.get('input[type="password"]').click();
+    cy.get('input[type="password"]').type('User1234');
```

Comment:
It's recommended to create a helper function for form filling to make your tests more maintainable and less repetitive. This function could take a form object as an argument and fill in the form based on the object's properties.